lib/tasks/demo.rake

```diff
@@ -869,4 +901,64 @@ namespace :demo do
   def create_fake_timestamp(from, to)
     Faker::Time.between_dates(from: from, to: to >= Date.today ? Time.now : to, period: :all)
   end
+
+  # Duplicate patient and all nested relations and change last name
+  def deep_duplicate_patient(patient, responder_id = nil)
+    new_pat = patient.dup
+    new_pat.responder_id = responder_id unless responder_id.nil?
+    new_pat.last_name = "#{Faker::Name.last_name}#{rand(10)}#{rand(10)}"
```

Comment:
do we want to make the `submission_token` unique as well?

Reply:
Very good point